Cargo.toml

```diff
@@ -23,6 +23,7 @@ debug-host-behaviour = []
 [dependencies]
 proxy-wasm = { git = "https://github.com/Kuadrant/proxy-wasm-rust-sdk.git", branch = "kuadrant" }
 serde_json = "1.0"
+gjson = "0.8"
```

Comment:
Let's not use that. For one, we have `serde_json` already (just above), but beyond that, [`gjson`](https://crates.io/crates/gjson) looks pretty dead wrt to maintenance even.